tests/cross/src/main/scala/tests/BaseDocumentHighlightSuite.scala

```diff
@@ -27,7 +27,7 @@ class BaseDocumentHighlightSuite extends BasePCSuite with RangeReplace {
       val highlights = presentationCompiler
         .documentHighlight(
           CompilerOffsetParams(
-            URI.create("file:/Highlight.scala"),
+            URI.create("file:///Highlight.scala"),
```

Comment:
Just asking, why this change?

Reply:
Not needed, there are some warning popping up and thought that might fix it, but it didn't. The warnings are not relevant, just pop up in tests.